README.md

```diff
@@ -1,2 +1,26 @@
 # semantic-html-refactor
-Refactor a marketing website (Horiseon) using semantic HTML and CSS
+
+## Description:
+
+Refactoring existing code without changing the design and functionality of the web application by mainly using semantic HTML and CSS.
+
+## Refactoring Tasks Completed:
+
+- Added an appropriate and descriptive document title
+- Utilized semantic-Html tags like main, aside, article etc in appropriate places
+- Fixed the bug that existed in the "Search Engine Optimization" nav-link
+- Added web accessibility elements and attributes like figcaption and alt in appropriate places
+- Changed the selector classes in CSS to target the correct refactored HTML tags
+- Combined numerous class selectors in CSS to reduce any duplication
+
+## Preview Image Of Code:
+
+![code-preview](./assets/screenshots/code-preview.PNG)
+
+## Preview Image Of Web-page:
+
+![web-page-preview](./assets/screenshots/web-page-preview.png)
+
+## Complete Project:
```

Comment:
Consider moving the link to deployed application towards the top of the README as it is the one of the first things that people visiting your repo will look for